### ISSUE TITLE
Ventilation feedback

### ISSUE THREAD

**Issue by richterdavid** [phpledge/badging#15]
Robust indoor ventilation should comply with ASHRAE 241's infection mitigation management mode's level of clean air supply, which is NN liters per second per person, with NN depending on the setting.

**Comment by bluesomewhere** [comment]
Yeah, it's high time to set some concrete targets here. One good habit I'm seeing emerge among some organizers is the use of CO2 monitors to identify issues and, as possible, address them, as well as to further fine tune their decision making for future events. I can imagine setting some targets using those as well, as they seem very in reach.

 Could I trouble you to provide a reference for the relevant ASHRAE targets?

**Comment by richterdavid** [comment]
Official copy: https://www.ashrae.org/technical-resources/standards-and-guidelines/read-only-versions-of-ashrae-standards
